pkg/plugin/v2/scaffolds/internal/templates/boilerplate.go

```diff
@@ -57,24 +86,25 @@ func (f *Boilerplate) SetTemplateDefaults() error {
 		return nil
 	}
 
-	// Pick a template boilerplate option
-	switch f.License {
-	case "", "apache2":
-		f.TemplateBody = apache
-	case "none":
-		f.TemplateBody = none
-	}
+	f.TemplateBody = boilerplateTemplate
 
 	return nil
 }
 
-const apache = `/*
+const boilerplateTemplate = `/*
 {{ if .Owner -}}
 Copyright {{ .Year }} {{ .Owner }}.
 {{- else -}}
 Copyright {{ .Year }}.
 {{- end }}
+{{ index .Licenses .License }}*/`
+
+var knownLicenses = map[string]string{
+	"apache2": apache2,
+	"none":    "",
```

Comment:
We have none test/check to ensure the none. Do you think that we could cover it with unit tests?

Reply:
We had tests for each file in the past but we removed them because they don't scale well.

Reply:
Hi @Adirio, 

The e2e tests have CI effort but the unit tests are not. Then, I cannot see how unit tests would not be scalable and I have no knowledge about we remove the unit tests in the past.I understand that the goal is to cover the project with the unit as much as possible. We actually added coveralls in the project in order to help us to improve its coverage see `coverage/coveralls — Coverage remained the same at 70.581%`.   


However, I tested it locally and shows that all worked fine. 

```
$ kubebuilder init --license=none
Writing scaffold for you to edit...
Get controller runtime:
$ go get sigs.k8s.io/controller-runtime@v0.6.3
Update go.mod:
$ go mod tidy
Running make:
$ make
/Users/camilamacedo/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
Next: define a resource with:
$ kubebuilder create api

$ cat hack/boilerplate.go.txt
/*
Copyright 2020.
```

So, IMO we can merge this one


/approve